aura.grammar

```diff
@@ -0,0 +1,56 @@
+
+use_statement: identifier ['as' identifier]
+
+function_def: 'fn' NAME '(' decl_list ')' type NEWLINE function_body
+
+return_statement: 'ret' [expression]
+
+decl_list: decl | decl ',' decl_list | epsilon
+
+type: i[1-9][0-9]* | f16 | f32 | f64 | f128 | '[' type ']' | '{' type ':' type '}'
+
+
+array : '[' identifier_list ']'
```

Comment:
We should group identifiers and constants into a `value` type and use it here. `[someVar, 1234i32, anotherVar]` is valid, but isn't reflected accurately since `identifier_list` doesn't include constants.